### PR TITLE
🪲 Fix bug related to ipynb conversion

### PR DESCRIPTION
**Pull request by FabienRoger**

### CODE REVIEW

File: opencodeblocks/graphics/edge.py [hunk 1]
```diff
@@ -252,10 +252,8 @@ def deserialize(self, data: OrderedDict, hashmap: dict = None, restore_id=True):
         self.path_type = data["path_type"]
         try:
             self.source_socket = hashmap[data["source"]["socket"]]
-            self.source_socket.add_edge(self, is_destination=False)
 
             self.destination_socket = hashmap[data["destination"]["socket"]]
-            self.destination_socket.add_edge(self, is_destination=True)
             self.update_path()
```
Comment (MathisFederico) [inline opencodeblocks/graphics/edge.py:257]: Won't this be a problem for copy & paste ?
Links need to be done with edges, I don't see where they are done now ...

**Review by MathisFederico — COMMENTED** [review]
A quick question about edges loading:

File: opencodeblocks/graphics/edge.py (hunk 1 above)
Comment (FabienRoger) [reply opencodeblocks/graphics/edge.py:257]: self.destination_socket = hashmap[data["destination"]["socket"]]
this uses a setter function which contains edge adding

**Review by MathisFederico — APPROVED** [review]
A nice bug-fix !
It would be nice to add a test to ensure this behave correctly in the future as well

**Comment by MathisFederico** [comment]
Also, in the future, add markers like this:
Fixes #128 

So that issue is linked and automatically closed

**Pull request merged**